Are we missing something with our install-from-source instructions

I am, for various reasons revolving around my laptop refusing to work, preferring instead to set itself on fire, using new-to-me machines. In this case I have a ubuntu 22.0.4 lts machine with minimal additions, and using micromamba for installation. With gxx, make, and bison installed (essentially what we tell users at https://sherpa.readthedocs.io/en/4.15.1/install.html#building-from-source) then I end up with errors related to libtool [actually, I don't know what the error is, but ut seems related to the libtool line]. The following is from the fftw extern build:

```
    Making install in fftw-3.3.10
    make[1]: Entering directory '/home/dburke/sherpa/sherpa-main/extern/fftw-3.3.10'
    Making install in support
    make[2]: Entering directory '/home/dburke/sherpa/sherpa-main/extern/fftw-3.3.10/support'
    make[3]: Entering directory '/home/dburke/sherpa/sherpa-main/extern/fftw-3.3.10/support'
    make[3]: Nothing to be done for 'install-exec-am'.
    make[3]: Nothing to be done for 'install-data-am'.
    make[3]: Leaving directory '/home/dburke/sherpa/sherpa-main/extern/fftw-3.3.10/support'
    make[2]: Leaving directory '/home/dburke/sherpa/sherpa-main/extern/fftw-3.3.10/support'
    Making install in kernel
    make[2]: Entering directory '/home/dburke/sherpa/sherpa-main/extern/fftw-3.3.10/kernel'
    /bin/bash ../libtool  --tag=CC   --mode=link gcc  -O3 -fomit-frame-pointer -mtune=native -malign-double -fstrict-aliasing -fno-schedule-insns   -o libkernel.la  align.lo alloc.lo assert.lo awake.lo buffered.lo cpy1d.lo cpy2d-pair.lo cpy2d.lo ct.lo debug.lo extract-reim.lo hash.lo iabs.lo kalloc.lo md5-1.lo md5.lo minmax.lo ops.lo pickdim.lo plan.lo planner.lo primes.lo print.lo problem.lo rader.lo scan.lo solver.lo solvtab.lo stride.lo tensor.lo tensor1.lo tensor2.lo tensor3.lo tensor4.lo tensor5.lo tensor7.lo tensor8.lo tensor9.lo tile2d.lo timer.lo transpose.lo trig.lo twiddle.lo  -lm
    libtool: link: false cru .libs/libkernel.a  align.o alloc.o assert.o awake.o buffered.o cpy1d.o cpy2d-pair.o cpy2d.o ct.o debug.o extract-reim.o hash.o iabs.o kalloc.o md5-1.o md5.o minmax.o ops.o pickdim.o plan.o planner.o primes.o print.o problem.o rader.o scan.o solver.o solvtab.o stride.o tensor.o tensor1.o tensor2.o tensor3.o tensor4.o tensor5.o tensor7.o tensor8.o tensor9.o tile2d.o timer.o transpose.o trig.o twiddle.o
    make[2]: *** [Makefile:419: libkernel.la] Error 1
    make[2]: Leaving directory '/home/dburke/sherpa/sherpa-main/extern/fftw-3.3.10/kernel'
    make[1]: *** [Makefile:710: install-recursive] Error 1
    make[1]: Leaving directory '/home/dburke/sherpa/sherpa-main/extern/fftw-3.3.10'
    make: *** [Makefile:386: install-recursive] Error 1
```

If you install fftw3 from conda-forge and set `setupcfg` to use that then the error gets moved to grplib:

```
    mv -f .deps/libgrp_la-grplib.Tpo .deps/libgrp_la-grplib.Plo
    /bin/bash ../libtool  --tag=CC   --mode=link gcc -DSTUB_ERROR_LIB=1  -g -O2 -static  -o libgrp.la -rpath /home/dburke/sherpa/sherpa-main/build/lib libgrp_la-grplib.lo libgrp_la-grp_priv.lo libgrp_la-stub_dserr.lo
    libtool: link: false cru .libs/libgrp.a  libgrp_la-grplib.o libgrp_la-grp_priv.o libgrp_la-stub_dserr.o
    make[2]: *** [Makefile:423: libgrp.la] Error 1
    make[2]: Leaving directory '/home/dburke/sherpa/sherpa-main/extern/grplib-4.9/src'
    make[1]: *** [Makefile:397: install-recursive] Error 1
    make[1]: Leaving directory '/home/dburke/sherpa/sherpa-main/extern/grplib-4.9'
    make: *** [Makefile:386: install-recursive] Error 1
    error: subprocess-exited-with-error
```

So, what is missing on my system (and we need to tell users they need)?
```

--- a/docs/install.rst
+++ b/docs/install.rst
@@ -141,11 +141,14 @@ Prerequisites
 The prerequisites for building from source are:
 
 * Python versions: 3.9 to 3.11
-* Python packages: ``setuptools``, ``numpy``
+* Python packages: ``setuptools``, ``numpy`` (these should be
+  automatically installed by ``pip``)
 * System: ``gcc`` and ``g++`` or ``clang`` and ``clang++``, ``make``, ``flex``,
-  ``bison`` (the aim is to support recent versions of these
-  tools; please report problems to the
-  `Sherpa issue tracker <https://github.com/sherpa/sherpa/issues/>`_).
+  ``bison``, ``ar`` (which may be provided by the ``binutils`` package)
+
+The aim is to support recent versions of these tools and libraries;
+please report problems to the
+`Sherpa issue tracker <https://github.com/sherpa/sherpa/issues/>`_.
 
 It is *highly* recommended that `matplotlib` and `astropy` be installed
 before building Sherpa, to avoid skipping a number of tests in the

